packages/core/package.json

```diff
@@ -23,9 +23,9 @@
     "build:ts:watch": "del ./dist/*.tsbuildinfo && tsc --p ./tsconfig.spec.json -w",
     "build:ts:spec": "tsc --b ./tsconfig.spec.json --force",
     "build:src": "npm-run-all build:sass:components build:ts",
-    "prebuild:api": "cd dist/core && yarn link",
-    "build:api": "wca analyze './dist/core/**/*.{js,ts}' --silent --format=json --outFile ./dist/core/custom-elements.json",
-    "postbuild:api": "cd dist/core && yarn unlink --no-save @clr/core",
+    "prebuild:api": "cd dist/core && yarn link && yarn link @clr/core",
+    "build:api": "wca analyze './dist/core/**/*.ts' --silent --format=json --outFile ./dist/core/custom-elements.json",
+    "postbuild:api": "cd dist/core && yarn unlink @clr/core && yarn unlink --no-save @clr/core",
```

Comment:
This fixed the issue of the API tool not understanding inherited classes since it only processed local import paths.